tests/tools/oecert/enc/enc.cpp

```diff
@@ -112,6 +114,55 @@ oe_result_t get_tls_cert_signed_with_key(
     return result;
 }
 
+oe_result_t get_plugin_evidence(
+    uint8_t* evidence,
+    size_t evidence_size,
+    size_t* evidence_out_size,
+    uint8_t* endorsements,
+    size_t endorsements_size,
+    size_t* endorsements_out_size)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    uint8_t* local_evidence = NULL;
+    size_t local_evidence_size = 0;
+    uint8_t* local_endorsements = NULL;
+    size_t local_endorsements_size = 0;
+
+    static const oe_uuid_t _ecdsa_uuid = {OE_FORMAT_UUID_SGX_ECDSA};
+
+    OE_CHECK(oe_attester_initialize());
```

Comment:
Same question about why this initialization is needed, it should be done inside oe_get_evidence() if not already done.

Reply:
Yes, I think you're right, so I added that to oecert so I can test it. Is the idea that all plugins will be initialized automatically or only the SGX ones (but, e.g., not the EEID plugin)?

Reply:
Turns out that's not the case; I need both `oe_verifier_initialize` and `oe_attester_initialize`, without them I get `OE_NOT_FOUND` errors. I think the design document says that the plugins should be initialized/registered automatically, but currently only `oe_verify_report` does so and `oe_verify_evidence` doesn't.

Reply:
For good practice of the API, please call `oe_verifier_initialize()` and/or `oe_attester_initialize()` and call only once.

Reply:
Thanks for clarifying @yentsanglee. I moved the verifier initialization in host_verify so that it's only done once.